src/pages/collections/_type/_.vue

```diff
@@ -207,6 +212,9 @@
         }
         return false;
       },
+      userIsEditor() {
+        return this.$store.state.auth.user && this.$store.state.auth.user.resource_access.entities && this.$store.state.auth.user.resource_access.entities.roles.includes('editor');
```

Comment:
extend this with a check to see if a user has the editor role for both entities and usersets

Reply:
Done